docs/roadmap.md

```diff
@@ -0,0 +1,41 @@
+# 2019 Roadmap for druid / piet
+
+There's been a lot of activity in GUI and 2D graphics, but it's not easy to make sense of it all. This document tries to explain what's in place, what the plans are for the next few months, and also a grander dream of what it might become.
+
+## Smaller modules composed in layers
+
+Building a GUI is a large-scale task, and to make any headway it needs to be broken into smaller components. I've factored this work into a number of Rust crates, largely so that each can be worked on and improved separately.
+
+In addition, many of these crates are designed as common infrastructure, and could be used by other GUI efforts.
+
+### druid
+
+### druid-shell
+
+The current scope of druid-shell is similar to that of winit: it pops up a window, and takes inputs. There is one additional set of functionality: it gives access to platform menus.
+
+I didn't use winit because I wanted to experiment, and didn't know whether design decisions of winit would limit performance or functionality. Shortly I want to take a close look at those questions. I can imagine using winit for that common functionality and layering other functionality on top.
```

Comment:
Some thoughts on what additional backends you'd like to support in the short and medium term would be nice